backend/tests/test_cinq_auditor_aws_required_tags_rds.py

```diff
@@ -0,0 +1,133 @@
+import datetime
+import random
+import uuid
+
+from cloud_inquisitor.constants import AuditActions
+from cloud_inquisitor.plugins.types.resources import RDSInstance
+from tests.libs.cinq_test_cls import MockRequiredTagsAuditor
+from tests.libs.lib_cinq_auditor_aws_required_tags import prep_rds_testing, VALID_TAGS, STANDARD_ALERT_SETTINGS_STOP
+from tests.libs.util_cinq import setup_test_aws
+from tests.libs.util_db import create_resource
+
+
+def test_rds_generic(cinq_test_service):
+    """
+    Test will pass if for an S3 bucket meet the following condition:
+        - Bucket is empty
+        - No Bucket Policy was set
+        - No Lifecycle Policy was set
+        - No tag was set
+
+    The Auditor will:
+        - Detect non-compliant S3 buckets
+        - Respect grace period settings
+        - Be able to remove an empty bucket successfully when the "REMOVE" criteria are met
+    """
+    # Prep
+    setup_info = setup_test_aws(cinq_test_service)
+    recipient = setup_info['recipient']
+    account = setup_info['account']
+    compliant_resources = []
+    non_compliant_resources = []
+
+    prep_rds_testing(cinq_test_service)
+
+    # Add resources
+    num_resources = 50
+
+    for i in range(0, num_resources):
+        compliance_selector = random.randint(0, 1)
+        if compliance_selector:
+            tags = VALID_TAGS
+        else:
+            tags = {}
+
+        new_resource = create_resource(
+            resource_type=RDSInstance,
+            resource_id=uuid.uuid4().hex,
```

Comment:
uuid.uuid4().hex is probably ok for resource_id though in future we might want to consider something in the same format that AWS uses.

Reply:
I can be wrong but afaik RDS id is different from EC2 ids (e.g. i-xxxxxxxx) and is made by users

So I guess `uuid.uuid4().hex` can fit in this case?